src/main/java/seedu/address/commons/util/DateUtil.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.commons.util;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_DATE_FORMAT;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class DateUtil {
+
+    private static final DateTimeFormatter DATE_PARSER;
+
+    private static final DateTimeFormatter DEFAULT_FORMATTER = DateTimeFormatter.ofPattern("dd MMM yyyy");
+
+    static {
+        String[] patterns = new String[]{"dd-MM-yyyy"};
```

Comment:
Perhaps we could make `dd-MM-yyyy` a constant string, since it is also used in line 40